lib/src/modules/apply.cc

```diff
@@ -153,19 +153,19 @@ begin
   env = Puppet.lookup(:environments).get('production')
```

Comment:
This is what is being replaced. If we move forward w/this we need to delete this line.